packages/m2m-gateway/src/clients/clientsProvider.ts

```diff
@@ -63,6 +72,15 @@ export type PagoPAInteropBeClients = {
 };
 
 export function getInteropBeClients(): PagoPAInteropBeClients {
+  const consumerDelegationsClient = patchZodiosClient(
+    delegationApi.createConsumerApiClient(config.delegationProcessUrl)
+  );
+  consumerDelegationsClient.use(zodiosMetadataPlugin());
+
+  const delegationClient = patchZodiosClient(
+    delegationApi.createDelegationApiClient(config.delegationProcessUrl)
+  );
+  delegationClient.use(zodiosMetadataPlugin());
```

Comment:
We could find a way to generalize this instead of having to patch manually each one of these clients